helix-view/src/document.rs

```diff
@@ -1713,6 +1713,12 @@ impl Document {
         self.version
     }
 
+    pub fn supports_path_completion(&self) -> bool {
```

Comment:
this seem like the wrong name, it should be `path_completions_enabled()` (or similar)